### ISSUE TITLE
Lots of apps not uninstallable, add an "run as Admin always" option?

### ISSUE THREAD

**Issue by firefoxlover** [Klocman/Bulk-Crap-Uninstaller#447]
1. This app is awesome! it is so needed, it allows automation and so many useful features, really nice!

But compared to Geek Uninstaller, it just doesnt work.

I think the problem is, that it doesnt run as root (Linux user, not much idea of Windows), but opens admin Powershell instances (or how the hell thats called) for every uninstall.

The result is, that many apps are not removable:
- Edge (Update works)
- other components

Can you just run the Program as admin? A feature to do that would be really good, even though of course it has security risks.

**Comment by Klocman** [comment]
BCU already runs as admin only, and all commands/applications it runs also run as admin as a result. There's no easy way to gain even higher privileges, it would require adding a system service or messing with system tasks, both of which I'd imagine can have compatibility issues on Linux, on top of allowing some very dangerous actions that would potentially let users destroy their OS.

Running as system wouldn't actually help to remove Edge since MS blocked that at some point. You need to run as TrustedInstaller specifically, which isn't able to access anything outside of these modern apps or critical system files, so it can't be used for BCU as a whole. It might be possible to run the PS scripts as this user but it needs testing and research to tell for sure, MS continually blocks any ways to remove these apps that people find.

**Comment by firefoxlover** [comment]
Thanks for the explanation! I wonder how Geek uninstaller does it, it also shows less programs though.

**Comment by Klocman** [comment]
From what I've seen other app uninstallers just don't list these entries, yeah.

**Comment by firefoxlover** [comment]
yes, Geek just doesnt show it. Must have been a recent update, weird. Didnt know Windows was still actively working at making Win10 suck. Thanks for your time!

**Comment by firefoxlover** [comment]
[Here is an article about how to remove Edge](https://woshub.com/uninstall-disable-microsoft-edge/)

```
$EdgeVersion = (Get-AppxPackage "Microsoft.MicrosoftEdge.Stable" -AllUsers).Version

# only recent one

$EdgeLstVersion=$EdgeVersion[-1]

# path to install dir

$EdgeSetupPath = ${env:ProgramFiles(x86)} + '\Microsoft\Edge\Application\' + $EdgeLstVersion

cd $EdgeSetupPath

.\Installer\setup.exe --uninstall --system-level --verbose-logging --force-uninstall
```

remove Webview too:
```
# determine location (probably not that one)

cd "C:\Program Files (x86)\Microsoft\EdgeWebView\Application\103.0.1204.37\Installer\ 

setup.exe --uninstall --msedgewebview --system-level --verbose-logging
```

Avoid Edge being reinstalled on Windows update, open Powershell

```
et-item -Path "HKLM:\Software\Microsoft\EdgeUpdate"
Set-ItemProperty -Path "HKLM:\Software\Microsoft\EdgeUpdate" -Name DoNotUpdateToEdgeWithChromium -Type "DWORD" -Value 1 –Force
```

This creates a Registry entry.

**Comment by chinanoahli** [comment]
> ndows connections with SimpleWall and thats it. No updates for me, sorry Microsoft.

Should `et-item -Path "HKLM:\Software\Microsoft\EdgeUpdate"` be `Set-item -Path "HKLM:\Software\Microsoft\EdgeUpdate"`?

**Comment by firefoxlover** [comment]
lol yes looks so XD thanks

**Comment by goyalyashpal** [comment]
> Didnt know Windows was still actively working at making Win10 suck. Thanks for your time!
> \- @ firefoxlover at https://github.com/Klocman/Bulk-Crap-Uninstaller/issues/447#issuecomment-1439957338

welcome to corporate produce ye' coming from peaceful linux land

wait, why do u love firefox lol - the company financed & stuffed by google. use [librewolf](https://librewolf-community.gitlab.io/) 😆
